pkg/storage/couchdb/couchdbstore.go

```diff
@@ -143,29 +142,23 @@ func (c *CouchDBStore) Put(k string, v []byte) error {
 		valueToPut = wrapTextAsCouchDBAttachment(v)
 	}
 
-	if c.revIDs[k] != "" {
-		var m map[string]interface{}
-		if err := json.Unmarshal(valueToPut, &m); err != nil {
-			return fmt.Errorf("failed to unmarshal put value: %w", err)
-		}
-
-		m["_rev"] = c.revIDs[k]
-
-		var err error
-		valueToPut, err = json.Marshal(m)
+	revID, err := c.getRevID(k)
```

Comment:
So this means that you want to overwrite the latest revision, regardless of what you read originally with Get. Is this what is desired?

Reply:
yes